dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -320,6 +320,8 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 	const showTopicFilterBank =
 		CAPIArticle.config.switches.automaticFilters && showKeyEventsCarousel;
 
+	const showToggle = !showTopicFilterBank || !CAPIArticle.availableTopics;
```

Comment:
Would it make things clearer if we centralised these decisions into one place and then had one decision outcome? A bit like

```ts
const decideWhichFilterToShow = ( ...things ): 'topics' | 'keyevents' => {
    ...logic
};

const filterToShow = decideWhichFilterToShow( ...things );
```

and then later

```ts
<div id="maincontent" css={bodyWrapper}>
	{CAPIArticle.keyEvents.length && filterToShow === 'keyevents' ? ( ... )}
</div>
```

Reply:
@oliverlloyd - this definitely would but we are actually going to be removing the key events carousel switch which will reduce the complexity around the logic here (PR to follow). As this is a hotfix, for now we will go ahead as is, but I like this pattern for future switch coupling!